zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/client/ObjectRecord.kt

```diff
@@ -16,18 +16,21 @@
 
 package com.ritense.objectenapi.client
 
+import com.fasterxml.jackson.annotation.JsonFormat
+import com.fasterxml.jackson.databind.JsonNode
 import java.time.LocalDate
 
 class ObjectRecord(
-    val index: Int?,
+    val index: Int? = null,
     val typeVersion: Int,
-    val data: Map<String, Any?>?,
-    val geometry: ObjectGeometry?,
+    val data: JsonNode? = null,
+    val geometry: ObjectGeometry? = null,
+    @JsonFormat(pattern = "yyyy-MM-dd")
```

Comment:
i'm still not sure why we would need this dateformat here. i think we should look into this.

Reply:
After changing the format, the test failed. When removing the annotation, no problem. It seems it is not needed.